"npm run start" missing script

### Describe the bug

[README.md](https://github.com/evcc-io/evcc/blob/master/README.md)

[UI development](https://github.com/evcc-io/evcc#ui-development)
npm run start

missing script

Kann es sein das hier "npm run dev" genutzt werden soll?

### Steps to reproduce

1.
2.
3.
...


### Configuration details

```yaml
nicht relevant
```


### Log details

```text
npm run start
npm ERR! Missing script: "start"
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run
```


### What type of operating system are you running?

Linux

### Version

_No response_

--- a/README.md
+++ b/README.md
@@ -61,7 +61,7 @@ For frontend development start the Vue toolchain in dev-mode. Open http://127.0.
 
 ```sh
 npm install
-npm run start
+npm run dev
 ```
 
 ### Integration tests

